### PR TITLE
AFM: Fix for a small problem pointed out by a user

### PR DESCRIPTION
**Pull request by rddheath**

### CODE REVIEW

File: modules/ROOT/pages/bat-example-test-suite.adoc [hunk 1]
```diff
@@ -69,7 +69,7 @@ If you write a second test, this one named `second-test.dwl`, you add this line
   file: api-tests/second-test.dwl
 ----
 
-The order in which you list the tests determines the order in which the BAT CLI runs them. In this example, the test `prodInfo.dwl` runs before the test `shoppingCart.dwl`:
+The order in which you list the tests determines the order in which the BAT CLI runs them. In this example, the test `orderFullfillment.dwl` runs before the test `shoppingCart.dwl`:
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/bat-example-test-suite.adoc]: Either Fulfillment or Fulfilment are correct words, but not Fullfillment.
If this is not an actual file that can be download and is an illustrative example, I'd suggest rewording all instances to **orderFulfillment.dwl**

**Review by Cristian-Venticinque — APPROVED** [review]

File: modules/ROOT/pages/bat-example-test-suite.adoc (hunk 1 above)
Comment (rddheath) [reply modules/ROOT/pages/bat-example-test-suite.adoc]: Yeah, I'll fix it here in this topic, but the developers need to do the same. I'll let them know.
